modules/tinymce/src/themes/silver/main/ts/modes/Inline.ts

```diff
@@ -78,7 +78,7 @@ const setupEvents = (editor: Editor, targetElm: SugarElement, ui: InlineHeader,
   // When the page has been scrolled, we need to update any docking positions. We also
   // want to reposition all the Ui elements if required.
   let lastScrollX = 0;
-  const updateUi = Throttler.last(() => ui.update(), 33);
+  const updateUi = Throttler.last(() => ui.update(Docking.refresh), 33);
```

Comment:
This merged before #8428 was merged, and later in 9414 we have made changes to `ui.update` to call `Docking.reset`, changing it to what it was before. 

https://github.com/tinymce/tinymce/blob/4c7c3d133a1789e797efe80d2f5a91c94adef2c3/modules/tinymce/src/themes/silver/main/ts/modes/Inline.ts#LL65C48-L65C48